package.json

```
@@ -109,6 +112,7 @@
     "eslint-plugin-node-dependencies": "^0.8.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-regexp": "^1.0.0",
+    "eslint-plugin-svelte": "file:.",
```

```suggestion
    "eslint-plugin-svelte": "link:."
```

A soft like should more suitable for this case, and we can preserve it always.

---

If we specify a directory in the `file:` URL, I think a symbolic link will be used.
However, it may be different depending on the OS. It seems that symbolic links are used, at least on MacOS.
`link:.` didn't seem to work on my local npm. Does it use a different package manager?

---

A constant development environment is very useful and necessary IMO.

---

You're right in team development. It makes sense to make the consistent development environment for this project.
Previously, I was the only one maintaining the project, so I wanted to prioritize giving the package manager a free choice for anyone trying to open a pull request, but that's no longer necessary.